plugins/modules/fabric_virtual_networks_workflow_manager.py

```diff
@@ -0,0 +1,2950 @@
+                    vn_in_ccc = self.get_vn_details_from_ccc(vn_name)
+                    vn_id = vn_in_ccc.get("id")
+                    self.delete_layer3_virtual_network(vn_name, vn_id).check_return_status()
+                else:
+                    self.log("Given Virtual network '{0}' is not present in Cisco Catalyst Center.".format(vn_name), "INFO")
+                    self.absent_vn.append(vn_name)
+                    continue
+
+        # Need ID of the anycast gateway to edelete the anycast gateway
+        anycast_gateways = config.get("anycast_gateways")
+        if anycast_gateways:
+            for anycast in anycast_gateways:
+                vn_name = anycast.get("vn_name")
+                ip_pool_name = anycast.get("ip_pool_name")
+                site_name = anycast.get("fabric_site_location").get("site_name_hierarchy")
+                site_id = self.get_site_id(site_name)
+                fabric_type = anycast.get("fabric_site_location").get("fabric_type")
+
+                if fabric_type == "fabric_site":
+                    fabric_id = self.get_fabric_site_id(site_name, site_id)
+                else:
+                    fabric_id = self.get_fabric_zone_id(site_name, site_id)
+
+                # Collect the gateway id with combination of vn_name, ip_pool_name and fabric id
+                unique_anycast = vn_name + "_" + ip_pool_name + "_" + site_name
+                anycast_details_in_ccc = self.get_anycast_gateway_details(vn_name, ip_pool_name, fabric_id)
+
+                if not anycast_details_in_ccc:
+                    self.absent_anycast.append(unique_anycast)
+                    self.log("Given Anycast gateway '{0}' is not present in Cisco Catalyst Center.".format(unique_anycast), "INFO")
+                    continue
+                gateway_id = anycast_details_in_ccc.get("id")
+
+                self.delete_anycast_gateway(gateway_id, unique_anycast).check_return_status()
+
+        return self
+
+    def verify_diff_merged(self, config):
+        """
+        Verify the addition/update status of fabric Vlan, layer3 Virtual Networks and
+        Anycast Gateway(s) in teh Cisco Catalyst Center.
+        Parameters:
+            self (object): An instance of a class used for interacting with Cisco Catalyst Center.
+            config (dict): The configuration details to be verified.
+        Returns:
+            self (object): An instance of a class used for interacting with Cisco Catalyst Center.
+        Description:
+            This method verifies whether the specified configurations have been successfully added/updated
+            in Cisco Catalyst Center as desired.
+        """
+
+        self.get_have(config)
+        self.log("Current State (have): {0}".format(str(self.have)), "INFO")
+        self.log("Desired State (want): {0}".format(str(self.want)), "INFO")
+
+        # Verify the creation/updation of fabric Vlan in the Cisco Catalyst Center
```

Comment:
Orwe can simply pass config and get the value inside the APIs and put all log messages inside the APIs.

```
self.verify_fabric_vlan(config)
self.verify_virtual_network(config)
self.verify_anycast_gateway(config)
```